src/components/pages/TopPage/hooks.ts

```diff
@@ -24,10 +24,10 @@ export default () => {
   }, [isAuthenticated, navigate, currentTeam, setLocalState, data, teamId]);
 
   useEffect(() => {
-    if (authError) {
+    if (authError || (!loading && data?.me === null)) {
       logout();
```

Comment:
In the future, I would like to display an error message after logging out, because currently it logs out silently.